docs/_articles/Power-Production-Distribution-and-Usage-Overview.md

```diff
@@ -0,0 +1,46 @@
+ 
+* Geothermal: Geothermal can only be utilized at favorable locations at the moment due to the cost of drilling and it would not be cost effective to use on a small scale. As drilling technology advances and the costs of fossil fuels goes up this will be used by large industrial applications in locations that didn’t used to be possible. As fracking has shown we are still learning the consequences of drilling into the earth so its use should be done with caution.
+
+* Solar: Solar energy is utilized in five ways; hydroelectric, biomass, solar thermal, wind and photovoltaics.
+
+ * Hydroelectric: Hydroelectric is a renewable energy source, however how it is currently implemented it causes damage to the ecosystems of rivers and floods land that could be used in other ways. There are ways of harnessing the power of a river without damming it. There are people who are lucky enough to have a stream on their property who do small scale hydroelectric. There are lots of issues the world faces regarding water. Some argue it’s more a pressing issue than energy so we need to be careful in how we use it. See this [Smithsonian article about the Colorado river: the colorado river runs dry]( http://www.smithsonianmag.com/science-nature/the-colorado-river-runs-dry-61427169/?no-ist)
+
+ * Biomass: Biomass is plants and algae that utilize photosynthesis. Biomass is very renewable but only if done carefully. Ethanol produced from corn is not a good choice because by using a food crop you are taking it from the mouth of a hungry person. The other issue is the energy required to make the fertilizer, harvest the corn process it into ethanol then transport it to where it’s needed. Other forms such as methane from landfills or from sewage treatment plants make sense as they are preventing the methane from getting into the air, they make power doing it and don’t use any new land in their construction. These biodigesters can also be easily scaled to that of the individual home, in fact traditional septic systems are biodigesters that do not trap and utilize the methane, water and nutrients produced. The other important form of biomass energy production is biochar. Biochar is the pyrolysis of organic matter to release the volatile gases that can be used as fuel and a byproduct of high quality charcoal. The charcoal can be then "seasoned" and mixed with soil to create Terra Prata, a highly productive soil. This form of carbon is very stable and so is a way of sequestering carbon in the soil for hundreds and possibly thousands of years. 
+ 
+ * Solar thermal: Solar thermal can be either unconcentrated or concentrated. 
+     
```

Comment:
And also, the image should go here, also explain the image rather than just putting it out there, people don't know what they're looking at.